src/App.css

```diff
@@ -346,29 +346,64 @@
   box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
   padding: 12px 16px;
   margin-bottom: 8px;
-  cursor: grab;
   border: 1px solid var(--border-color);
   color: var(--text-primary);
   font-size: 14px;
   transition: all 0.3s ease;
   display: flex;
-  justify-content: space-between;
   align-items: center;
   gap: 12px;
+  cursor: grab;
+  user-select: none;
+}
+
+.task:active {
+  cursor: grabbing;
+}
+
+.task.dragging {
+  background: var(--hover-bg);
+  box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
+  opacity: 0.9;
+  transform: scale(1.02);
+  cursor: grabbing;
+}
+
+.task-list.dragging-over {
+  background: var(--hover-bg);
+  border-radius: 4px;
+  padding: 8px;
+  transition: all 0.2s ease;
 }
 
 .task:hover {
   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-  border-color: var(--border-color);
+  border-color: var(--primary-color);
   background: var(--hover-bg);
 }
 
-.task:active {
-  cursor: grabbing;
+.task-drag-handle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px;
+  border-radius: 4px;
+  color: var(--text-secondary);
+  transition: all 0.2s ease;
+}
+
+.task:hover .task-drag-handle {
+  color: var(--primary-color);
+}
+
+.task.dragging .task-drag-handle {
+  color: var(--primary-color);
 }
 
 .task-content {
   flex: 1;
+  min-width: 0;
+  pointer-events: none;
 }
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
.task-content {
  flex: 1;
  min-width: 0;
  pointer-events: none;
}

.task-content a,
.task-content button {
  pointer-events: auto;
}
```
The task-content class has pointer-events: none which might affect nested interactive elements. Consider using a more specific selector to maintain clickability of important nested elements.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;